Freshli/Runner.cs

```diff
@@ -4,6 +4,12 @@
 
 namespace Freshli {
   public class Runner {
+
+    private static readonly string SaveResultsToFile =
+      Environment.GetEnvironmentVariable("SAVE_RESULTS_TO_FILE") ?? "false";
```

Comment:
Environment variables aren't actually loaded anywhere...I still need to figure this part out, but it's just for development, so I didn't want to hold things up.

Reply:
Can something like [dotenv](https://github.com/bkeepers/dotenv) help?

Reply:
Yes, we use [DotNetEnv](https://github.com/tonerdo/dotnet-env) in `freshli-web`, but I haven't been able to get things to load cleanly here. I've created issues #181 to address this.